src/index.js

```diff
@@ -1,3 +1,137 @@
+// 전역 변수
+const todoItem = `
+<div class="view">
+  <input class="toggle" type="checkbox"/>
+  <label class="label">새로운 타이틀</label>
+  <button class="destroy"></button>
+</div>
+<input class="edit" value="새로운 타이틀" />
+`;
+let state = {
+    condition: "all",
+    items: [],
+};
+const todoList = document.getElementById("todo-list");
```

Comment:
```suggestion
const $todoList = document.getElementById("todo-list");
```
```suggestion
const todoListTarget = document.getElementById("todo-list");
```
이런식으로 돔 객체에는 `$`을 prefix로 사용하거나 `Target`을 suffix로 사용하는 것을 추천드립니다! todoList 라고 하면 보통 데이터를 연상하기 때문입니다